examples/beego/main.go

```diff
@@ -0,0 +1,133 @@
+package main
+
+import (
+	"encoding/json"
+	"errors"
+	"fmt"
+	"io"
+	"net/http"
+	"strings"
+	"time"
+
+	"github.com/airbrake/gobrake/v5"
+	beegobrake "github.com/airbrake/gobrake/v5/beego"
+	"github.com/beego/beego/v2/server/web"
+)
+
+type Date struct {
+	Date int64 `json:"date"`
+}
+
+type WeatherInfo struct {
+	Lat            float32       `json:"lat,omitempty"`
+	Lon            float32       `json:"lon,omitempty"`
+	Timezone       string        `json:"timezone,omitempty"`
+	TimezoneOffset float32       `json:"timezone_offset,omitempty"`
+	Current        interface{}   `json:"current,omitempty"`
+	Minutely       []interface{} `json:"minutely,omitempty"`
+	Hourly         []interface{} `json:"hourly,omitempty"`
+	Daily          []interface{} `json:"daily,omitempty"`
+}
+
+type Controller struct {
+	web.Controller
+}
+
+var ProjectId int64 = 363389
+var ProjectKey string = "baa755018d5e35e07897ee0087fcce9c"
```

Comment:
These are a **real** `ProjectId` and `ProjectKey`, pointing to [this](https://getexceptional.airbrake.io/projects/363389) project. We should replace these with the usual example values. Then maybe re-base the new commit into the previous one so these values aren't visible.

Reply:
Great catch, I will replace them. I will squash all the commits in `beego-integration` before merging into the `main` branch.

Reply:
Regenerate the project key.

Reply:
Yes